nexus/types/src/external_api/views.rs

```diff
@@ -1488,6 +1488,22 @@ pub struct TargetRelease {
 
     /// The source of the target release.
     pub release_source: TargetReleaseSource,
+
+    /// Whether the system has been updated to a new release through the
+    /// recovery (MUPdate) path since the target release was last set.
+    ///
+    /// In this case, the target release is considered to be no longer active,
+    /// and a new release must be set.
```

Comment:
```suggestion
    /// If true, indicates that at least one sled in the system has been updated through the recovery (MUPdate) path since the last time the target release was set.
    ///
    /// In this case, the system will ignore the currently-set target release, on the assumption that continuing an update may re-introduce or exacerbate whatever problem caused the recovery path to be used.  You must set the target release again in order to resume automated upgrades.
```

Feel free to edit or go back to yours but what I was trying to edit here was:

- it wasn't necessarily a _new_ release that we mupdated to
- elaborating on why the system ignores this value and what the operator needs to do to fix it